pages/_app.tsx

```diff
@@ -134,6 +136,11 @@ if (isMock()) {
   configureMockEnvironmentForTesting();
 }
 
+// Expose app feature flags to the protocol through an app-agnostic API.
+if (features.repaintEvaluations) {
+  setRepaintAfterEvaluations(true);
+}
```

Comment:
I assume this (during app initialization) is as good a place as any to set this logic?

Reply:
I'd probably lean towards putting it into `devtools.ts` with the rest of the recording-specific setup, but it'll work either way

Reply:
Thanks! 👍🏼 

Looking at `devtools.ts` I don't see a place that stands out to me as better for it but I don't feel strongly at all if we move it there (or somewhere else)